### DISCUSSION TITLE
Port 1 power inside USB packets

### DISCUSSION THREAD

**Discussion by Alex-888888** [jankae/LibreVNA#242] [Q&A] [answered]
Jan,
I'm playing with the Libre VNA and I'am connected to it with a small python software.
I apply a signal on port 1 and I view it on the Libre VNA GUI with almost the right level.
I check the fram with wireshark for my little software and for the Libre VNA GUI.
The extracted data for the same level is 3d25b213 (approximatively).
Libre VNA GUI display a level of -27.86dBm that is right.
When I convert the hexa packet I found -13.93dBm (with a measured power of 0.0404mW).
My calculation is : powerdB = 10 * log10(powerMw) (that's what I learn in my young days).
But it seems that the good calculation is powerdB = 20 * log10(powerMw).
I'm probably wrong somewhere or I haven't well understood your document (I'm french).
Thank you for your help

**Comment by jankae** [comment, marked as answer]
Hi Alex,
the transmitted ADC data over USB is a voltage, not a power. As a result, you need to use the 20*log10 formula.

**Comment by Alex-888888** [comment]
Thank,
After reviewing the document and with your explanations it's clear.
